requirements.txt

```diff
@@ -1,5 +1,5 @@
-azure-keyvault==1.0.0
-msrestazure==0.4.34
-adal==1.2.4
-pyopenssl==18.0.0
-kubernetes==12.0.1
+azure-keyvault==4.1.0
+msrestazure==0.6.4
+adal==1.2.7
+pyopenssl==22.0.0
+kubernetes==23.3.0
```

Comment:
since you're updating some major versions after a long time this project wasn't updated, what kind of tests have you done to make sure nothing is broken?